File: WarehouseSystem/WarehouseSystem/WbSaveDto.cs
using Newtonsoft.Json;
using System;

namespace WarehouseSystem
{
    class WbSaveDto
    {
        public string wbNo { get; set; }
        public string wbDate { get; set; }
        public int boxes { get; set; }
        public int pallets { get; set; }
        public string centerId { get; set; }

        public WbSaveDto (string wbNo,string wbDate,string boxes,string pallets,string centerId)
        {
            this.wbNo = wbNo;
            this.wbDate = wbDate;
            if (boxes == "")
                this.boxes = 0;
            else
                this.boxes = Convert.ToInt32(boxes);
            if (pallets == "")
                this.pallets = 0;
            else
                this.pallets = Convert.ToInt32(pallets);
            this.centerId = centerId;
        }

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}

File: warehouse-system/src/main/java/com/who/warehousesystem/dto/WbSaveDto.java
package com.who.warehousesystem.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor

public class WbSaveDto {

    private Integer wbNo;
    private LocalDate wbDate;
    private Integer boxes;
    private Integer pallets;
    private Integer centerId;
}
